### ISSUE TITLE
Rebuild on top of Rhymen/go-whatsapp

### ISSUE THREAD

**Issue by lieuwex** [lieuwex/whapp-irc#40]
https://github.com/Rhymen/go-whatsapp

ref #25

**Comment by poVoq** [comment]
Disadvantage might be that there is a higher risk of Facebook banning accounts for using 3rd party clients. Currently there seems very little chance that they can distinguish this bridge from a regular browser, right?

**Comment by lieuwex** [comment]
Yeah I was thinking about this too, so I'm not really sure if this is high priority for me since basically it works _good enough™_ and we can't miss any protocol updates or something like that.
